### DISCUSSION TITLE
how to ask github to sponser a hackathon?

### DISCUSSION THREAD

**Discussion by Mintype** [community/community#110569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i just want an email i can ask this to. any help is appreciated.

**Comment by Mintype** [comment]
i know its not directly related to "github sponsors"

**Comment by Mintype** [comment]
> [sami5555sami@gmail.com](mailto:sami5555sami@gmail.com)
Where did you get this email from?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @SmallPlayz , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by Mintype** [reply]
thanks
